autoendpoint/src/routers/webpush.rs

```diff
@@ -117,6 +123,17 @@ impl Router for WebPushRouter {
                 trace!("Response = {:?}", response);
                 if response.status() == 200 {
                     trace!("Node has delivered the message");
+                    self.metrics
+                        .time_with_tags(
+                            "notification.total_request_time",
+                            (notification.timestamp - autopush_common::util::sec_since_epoch())
+                                * 1000,
+                        )
+                        .with_tag("platform", "websocket")
+                        .with_tag("app_id", "direct")
+                        .with_tag("stored", &notification.stored.to_string())
```

Comment:
This is the only place one of the new `stored` fields is used, and I think it's inaccurate here? as the `store_notification` would have always been called by now and it doesn't modify this field on the `notification` object (only on the internal copy it makes).

I don't think we need or should probably have any of new `stored` fields -- especially on `DynamoDbNotification`, that type basically says the notification is stored (or will be stored shortly by the end of the request). It can become tricky/potentially buggy to keep accurate, feels like more the job of an ORM unless it's really necessary.

Reply:
It was originally added because I wanted to know if messages were getting dropped by storage. Adding the flag would at least indicate the source of the message being sent through. 

It's not super necessary and there might be ways to better address if we determine it's a point of loss.